### PR TITLE
Security fix: prevent auto-translation

### PR DESCRIPTION
**Pull request by sesam**
Ensure automatic translation services does not get a continuous copy of the page's contents

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
```
Comment (0perationPrivacy) [inline frontend/index.html:4]: <meta> tags are self closing tags in HTML5 and the trailing slash is optional. only XHTML or XML requires it.

File: frontend/index.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+    <meta name="google" content="notranslate" />
```
Comment (0perationPrivacy) [inline frontend/index.html:5]: this tag doesn't prevent google from translating anymore according to this:
https://stackoverflow.com/questions/55954813/meta-name-google-content-notranslate-not-working-anymore
I believe "content" needs to be replaced with "value". But I have not tested it. Will integrate this if you can confirm which one works.

File: frontend/index.html (hunk 1 above)
Comment (sesam) [reply frontend/index.html]: Thanks - I changed to follow this standard then. (A code reformatter would help here to keep a single shared style)

File: frontend/index.html (hunk 2 above)
Comment (sesam) [reply frontend/index.html]: Oh, I just copied this from a place where it's in use and working for us. After googling this, the 2021 way to do <html translate="no" according to https://stackoverflow.com/a/12238414/296639
I'll implement this addition also in the other app where I got the old code from :) Thanks!

**Pull request merged**